src/lib.rs

```diff
@@ -32,6 +32,7 @@ pub enum AcpiError {
     RsdpIncorrectSignature,
     RsdpInvalidOemId,
     RsdpInvalidChecksum,
+    NoValidRsdp,
```

Comment:
Does the spec specify whether this always means that ACPI is not supported by the hardware if we can’t find the RSDP, or can there be other reasons? If the former, a better error could be `AcpiNotSupported`

Reply:
I don't know what it means *exactly*, but we just return this when we have searched the whole of all areas without finding a valid rsdp.

Reply:
It could mean that the user is really running UEFI under *some* circumstances.